ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.component.ts

```diff
@@ -179,6 +180,10 @@ export class AddressInputComponent implements OnInit, OnDestroy {
   set addressErrors(addressErrors: AddressError[]) {
     this.displayVerificationErrors(addressErrors);
   }
+
```

Comment:
To make the `required` flag stick you will need to go a bit further and add the required validation to the form group (referenced in line 27 in this file and instantiated here: https://github.com/broadinstitute/ddp-angular/blob/2512fef4e0e0cdb25d1445cefb260043d7b786ce/ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.service.ts#L54